extensions/map-layers/src/ui/Interfaces.ts

```diff
@@ -41,7 +42,16 @@ export interface MapFeatureInfoPropertyGridOptions {
   isPropertySelectionEnabled?: boolean;
 }
 
+export type MapHitEvent = BeEvent<(hit: HitDetail) => void>;
+
 export interface MapFeatureInfoOptions {
+  /**
+   * HitDetail Event whenever the map is clicked.
+   * Typically the HitDetail object is provided by ElementLocateManager.doLocate.
+   * Every time this event is raised, FeatureInfoWidget will attempt to retrieve data from MapLayerImageryProviders.
+   */
+  onMapHit?: MapHitEvent;
```

Comment:
I think this should be mandatory, FeatureInfoWidget wont work without this event.  By making it optional, you might confuse developers.

Reply:
updated